docs/guides/UpgardeGuide-v5/Changes.md

```diff
@@ -0,0 +1,323 @@
+
+
+### ServeStaticService
+
+The serve static service can serve a static site or compile a site from the webpack config. The compiled site will be served by webpack-dev-server and the static site will be served by terra's serve static server.
+
+The service will no longer inject the locale into served html files so when serving static sites, you will be responsible for adding the locale to the static files.
+
+When compiling a sites with webpack, the ```defaultLocale```` environment variable will be passed to the webpack config for the current test locale. This will be done automatically for projects using terra-dev-site.
+
+For projects extending terra-toolkit's default webpack configuration, be sure to make the following changes to enable locale testing:
+
+```diff
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const merge = require('webpack-merge');
+
+// Import the terra-toolkit configuration.
+const defaultWebpackConfig = require('terra-toolkit/config/webpack/webpack.config');
+
+// Create the app-level configuration
+const appWebpackConfig = () => ({
+  entry: {
+    index: path.resolve(path.join(__dirname, 'lib', 'site', 'Index')),
+  },
+  plugins: [
+      new HtmlWebpackPlugin({
++       lang: defaultLocale,
+        template: path.join(__dirname, 'lib', 'index.html'),
+      }),
+    ],
+});
+
+// combine the configurations using webpack-merge
+const mergedConfig = (env, argv) => (
+  merge(defaultWebpackConfig(env, argv), appWebpackConfig(env, argv))
+);
+
+module.exports = mergedConfig;
+```
+
+Template:
+
+```html
+<!doctype html>
+<html lang="<%= htmlWebpackPlugin.options.lang %>" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>index</title>
+  </head>
+  <body>
+    <h1>index</h1>
+  </body>
+</html>
+```
+
+For compiled sites, the service will respect the devServer setting in webpack config with a few exceptions:
+
+```javascript
+module.exports = {
+  devServer: {
+    // Disable hot reloading and javascript injection to watch for changes.
+    hot: false,
+    inline: false,
+    host: '0.0.0.0',
+    publicPath: '/',
+    port, // From wdio config.
+    index, // From wdio config.
+    stats: {
+      colors: true,
+      children: false,
+    },
+  },
+};
+```
+
+### AxeService
+The AxeService has been removed. The accessibility capabilities the Axe Service provided has been move to the Terra Service because the services could not run independently. The `axe` key can still be used for configuration and the test helper and axe chai assertion can be used. Please note, the `runOnly` option has been removed from Terra.should.beAccessible test helper and axe chai method and resetScroll has been enabled.
```

Comment:
```suggestion
The AxeService has been removed. The accessibility capabilities the Axe Service provided has been moved to the Terra Service because the services could not be run independently of one another. The `axe` key can still be used for configuration and the test helper and axe chai assertion can be used. Please note, the `runOnly` option has been removed from Terra.should.beAccessible test helper and axe chai method and resetScroll has been enabled.
```